spinoso-string/src/lib.rs

```diff
@@ -1922,17 +1922,13 @@ impl String {
     /// use core::ops::Range;
```

Comment:
if we make the change to remove the range struct instantiation, let's remove this import in the doc test

Reply:
Oops! Thanks for catching this. Removed in the amended 681d619